docs/data/joy/main-features/color-inversion/color-inversion.md

```diff
@@ -41,88 +36,52 @@ The color inversion feature is only available for `soft` and `solid` variants be
 
 ### Trade-offs
 
-- If the surface component contains just a few components, the style sheet size of the CSS variables might be **bigger** than customizing the styles of each individual child.
+- If the surface component contains just a few components, the style sheet size of the CSS variables might be bigger than customizing the styles of each child.
 - It doesn't work with browsers that don't support [CSS variables](https://caniuse.com/css-variables).
 
 ## Usage
 
-### Surface components
+### Supported components
 
-Surface components, including the Alert, Card, Drawer, ModalDialog, Menu, and Sheet, have the `invertedColors` prop to enable color inversion for `solid` and `soft` variants.
+All the surface components, including others like the Alert, Card, Drawer, Modal Dialog, Menu, and Sheet, accept the `invertedColors` prop for their `solid` and `soft` variants.
 
 {{"demo": "ColorInversionSurface.js"}}
 
 ### Portal popup
 
-By default, color inversion has no effect on the popup slot of the Autocomplete, Menu, and Tooltip components.
-To enable it, set `disablePortal` to true on the `slotProps`.
-
-:::info
-The popup slot of the Select component has `disablePortal` set to true by default.
-:::
+Color inversion does not affect the popup slot of the Autocomplete, Menu, and Tooltip components by default.
+To enable it, on the `slotProps` prop of these components, set `disablePortal` to true.
 
 {{"demo": "ColorInversionPopup.js"}}
 
-### Skip color inversion on a child
-
-To skip color inversion on a specific child, set `data-skip-inverted-colors` attribute to the component.
+### Turning it off on a specific child
 
-The component with `data-skip-inverted-colors` and its children will be excluded from the color inversion.
+Add the `data-skip-inverted-colors` attribute to the child component you want to turn color inversion off within an inverted container.
```

Comment:
```suggestion
When `invertedColors` is applied to a parent, you can add the `data-skip-inverted-colors` attribute to a child to prevent that child from being inverted.
```